src/main/kotlin/no/nav/helse/prosessering/v1/PdfV1Generator.kt

```diff
@@ -210,3 +210,5 @@ private fun String.sprakTilTekst() = when (this.toLowerCase()) {
     "nn" -> "Nynorsk"
     else -> this
 }
+
+private fun MeldingV1.harFlereAktiveVirksomheter() = this.selvstendigVirksomheter.filter { it.tilOgMed == null }.size > 1
```

Comment:
Hvofor ikke bruke `it.harFlereAktiveVirksomheter` her?

Reply:
Ah den henger igjen fra da jeg ikke visste det var et eget felt. Skal fjernes

Reply:
👍